src/views/scans/showAggregateReportModal.tsx

```diff
@@ -59,7 +59,7 @@ const formatSortFilterReportStats = (
 ) => {
   const { results: resultStats = {}, diagnostics: diagnosticStats = {} } = report;
 
-  return [...(resultStats && Object.entries(resultStats)), ...(diagnosticStats && Object.entries(diagnosticStats))]
```

Comment:
Help me understand what went on here:  the function is not safe if either object is null or undefined ?

Reply:
So this was an actual bug in implementation that we corrected in subsequent review fix... It leverages `truthiness` and the explicit setting of a value

- `undefined` falls back to the default object, ie `const { results: resultStats = {}`
- `null` is an explicit value that overrides the default empty object

You can see the end results yourself in console... 

```
var testing = [...(null && Object.entries(null))];
console.log(testing);
// result error, `not iterable`
```

vs

```
var testing = [...Object.entries(null || {})];
console.log(testing);
// result empty array/list
```